docs/ref/forms/renderers.txt

```diff
@@ -123,6 +123,7 @@ Each widget has a ``template_name`` attribute with a value such as
 ``input.html`` by defining ``django/forms/widgets/input.html``, for example.
 See :ref:`built-in widgets` for the name of each widget's template.
 
-If you use the :class:`TemplatesSetting` renderer, overriding widget templates
+If you want to override widget templates, you must use the
```

Comment:
Doesn't "You can't override built-in widget templates using the other built-in renderers." convey the meaning? There's a non-zero cost with changing the wording since documentation translators will have to redo work.

Reply:
I was skimming the doc in an attempt to resolve my issue and therefore didn't read past the first sentence, heh. IMO, this should read as a call to action and the current wording isn't _strong_ enough to emphasise that this was something I *have* to do. I'd also considered moving this into a note (or even warning) admonition of some time, like I added in the previous file, but decided to take this route instead. If you're really against it then I can certainly drop this hunk, but I do think something needs to change here.